core/xml.js

```diff
@@ -349,6 +349,14 @@ Blockly.Xml.domToWorkspace = function(xml, workspace) {
       }
     } else if (name == 'shadow') {
       goog.asserts.fail('Shadow block cannot be a top-level block.');
+    }  else if (name == 'variables') {
+      if (i == 1) {
```

Comment:
Because this is saved to a file the first node ends up being <xml xmlns="http://www.w3.org/1999/xhtml">